client/me/help/help-contact-closed/index.jsx

```diff
@@ -11,187 +11,29 @@ import i18n, { localize } from 'i18n-calypso';
-				) }
-			</p>
-		</div>
-	);
-} );
-
-const BusinessPlanMessage = localize( ( { translate } ) => {
-	return (
-		<div>
-			<p>
-				{ Date.now() < closedStartDate
-					? translate(
-							'Live chat support will be closed from %(closed_start_date)s through %(closed_end_date)s, with the ' +
-								'exception of limited hours %(limited_hours_start_date)s–%(limited_hours_end_date)s. ' +
-								'Email support will be open during that time, and we will reopen live chat on %(support_resume_date)s.',
-							{
-								args: {
-									closed_start_date: closedStartDate.format( 'dddd, MMMM D' ),
-									closed_end_date: closedEndDate.format( 'dddd, MMMM D' ),
-									limited_hours_start_date: limitedHoursStartDate.format( 'MMMM D' ),
-									limited_hours_end_date: limitedHoursEndDate.format( 'D' ),
-									support_resume_date: supportResumeDate.format( 'dddd, MMMM D' ),
-								},
-							}
-						)
-					: translate(
-							'Private support will be closed through %(closed_end_date)s, included. ' +
-								'We will reopen private support on %(support_resume_date)s.',
-							{
-								args: {
-									closed_end_date: closedEndDate.format( 'dddd, MMMM D' ),
-									support_resume_date: supportResumeDate.format( 'dddd, MMMM D' ),
-								},
-							}
-						) }
-			</p>
-			<p>
-				{ translate(
-					'Live chat will be available on September 13, 14, and 15 between the hours of 10:00am-1:00pm PDT ' +
-						'and 2:30pm-5:00pm PDT.'
-				) }
-			</p>
-			<p>
-				{ translate(
-					'Why? Once a year, the WordPress.com Happiness Engineers and the rest of the WordPress.com family get together ' +
-						'to work on improving our services, building new features, and learning how to better serve you, our users. ' +
-						'But never fear! If you need help in the meantime, you can submit an email ticket through the contact form.'
-				) }
-			</p>
-		</div>
-	);
-} );
-
-const HelpContactClosed = ( { translate, sitePlanSlug } ) => {
-	let message;
-
-	// Show different messages based on the plan of the site requesting help
-	switch ( sitePlanSlug ) {
-		case PLAN_PERSONAL:
-		case PLAN_PREMIUM:
-			message = <PersonalAndPremiumPlanMessage />;
-			break;
-		case PLAN_BUSINESS:
-			message = <BusinessPlanMessage />;
-			break;
-		default:
-			message = <NonPlanPaidMessage />;
-			break;
-	}
+const closedStartDate = i18n.moment( 'Thu, 23 Nov 2017 00:00:00 +0000' );
+const closedEndDate = i18n.moment( 'Fri, 24 Nov 2017 00:00:00 +0000' );
 
+const HelpContactClosed = ( { translate } ) => {
 	return (
 		<div className="help-contact-closed">
-			<FormSectionHeading>
-				{ translate( 'Limited Support %(closed_start_date)s - %(closed_end_date)s', {
-					args: {
-						closed_start_date: closedStartDate.format( 'MMM D' ),
-						closed_end_date: closedEndDate.format( 'D' ),
-					},
-				} ) }
-			</FormSectionHeading>
-			{ message }
+			<FormSectionHeading>{ translate( 'Limited Support on Thanksgiving' ) }</FormSectionHeading>
```

Comment:
Hi! I've found a possible matching string that has already been translated **6** times:
`translate( 'Limited Support For Thanksgiving' )` *ES Score: 18*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).